core/components/paragraph.js

```diff
@@ -104,36 +104,6 @@ export function convertToParagraph() {
     setBlockType(getTypeFromSchema(state.schema))(state, dispatch);
 }
 
-export function jumpToStartOfParagraph() {
```

Comment:
lets still keep this function to avoid breaking change.

Reply:
done